### PR TITLE
DEVELOPER-1318 404s being returned from github during demo creation does...

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please

**Comment by LightGuard** [comment]
I have merged the change in aweplug, so you're welcome to update to the latest aweplug and update the pull request, and squash.

**Comment by LightGuard** [comment]
@paulrobinson This is actually the error we are expecting. I know it sounds backwards, but we were expecting this pull request to fail. We're good to merge this and also https://github.com/jboss-developer/jboss-developer-demos/pull/11

File: Gemfile.lock [hunk 1]
```diff
@@ -1,29 +1,26 @@
 GIT
   remote: git://github.com/LightGuard/awestruct.git
-  revision: 05bf2c7fc4027c00115b6bc2e9ff0ffcf3eacc8e
+  revision: 5dbe512680b7aa094598ecdcbb6e52d3e75aa32f
```
Comment (paulrobinson) [inline Gemfile.lock]: Did you mean to update Awestruct?

File: Gemfile [hunk 2]
```diff
@@ -28,9 +29,9 @@ gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
 gem 'aweplug', github: 'awestruct/aweplug'
-# gem 'aweplug', github: 'Dantheman720/aweplug', :branch => 'DEVELOPER-1241'
-#gem 'aweplug', path: '~/projects/ruby/aweplug'
-#gem 'aweplug', path: '../aweplug'
+# gem 'aweplug', github: 'Dantheman720/aweplug', :branch => 'DEVELOPER-1318'
```
Comment (paulrobinson) [inline Gemfile]: You seem to have updated the branch value, but left the line commented out.

**Comment by paulrobinson** [comment]
Replaced with: https://github.com/jboss-developer/www.jboss.org/pull/877

**Pull request closed without merging**